docs/api/en/math/Matrix4.html

```diff
@@ -138,8 +138,8 @@ <h3>[method:this copyPosition]( [param:Matrix4 m] )</h3>
 
 		<h3>[method:null decompose]( [param:Vector3 position], [param:Quaternion quaternion], [param:Vector3 scale] )</h3>
 		<p>
-		Decomposes this matrix into it's [page:Vector3 position], [page:Quaternion quaternion] and
-		[page:Vector3 scale] components.
+		Decomposes this matrix into it's [page:Vector3 position], [page:Quaternion quaternion] and [page:Vector3 scale] components.
+		The method can't be used to decompose an object's world matrix if its parent has a non-uniform scale.
```

Comment:
That statement is incorrect. You can't make such general statements based on a specific example.

It is better to say nothing.

Reply:
What would be a better description? Or do you think that it's better to not say anything and let users figure it out on their own?

Reply:
> You can't make such general statements

TBH, I did not have the impression that it's a general statement. It a specific note intended for `Object3D.matrixWorld`.

Reply:
@Mugen87 As I said, the statement is not true.

A simple counterexample is a non-uniformly scaled parent matrix with a child having the Identity as its matrix. The child's world matrix is decomposable in that case. There are infinitely many other examples.

@mrdoob I would say nothing. If you insist on saying something, you could say "Note: not all matrices can be decomposed in this manner."

Reply:
A bit of a delay but we finally got to discuss the statement "off-line".

@WestLangley suggested this one which I find educative:

```
Note: Not all matrices are decomposable in this way. For example, if an object has a non-uniformly scaled parent, then the object's world matrix may not be decomposable, and this method may not be appropriate.
```

I'll update the file directly.

Reply:
https://github.com/mrdoob/three.js/commit/3c87cc7bb7ce626d23102597d222543f216c3bf6